### ISSUE TITLE
Firefox PWA

### ISSUE THREAD

**Issue by roughnecks** [fr0tt/benotes#99]
Hello,

can you add a proper PWA for Firefox Mobile? At the moment I can only add a shortcut to the homepage.

Thanks

**Comment by fr0tt** [comment]
Hi again,

I already noticed that, but to be honest, I have no idea why it's not working. 
There should be no difference between browsers when it comes to installing PWAs. Yet it works with at least all chromium based browsers on mobile and desktop, but not with Firefox.

**Comment by roughnecks** [comment]
yeah, I don't know either but thanks for looking into it.
Also feel free to close this issue if you think nothing else can be done.

Thanks

**Comment by Systemsemaphore** [comment]
> I already noticed that, but to be honest, I have no idea why it's not working. There should be no difference between browsers when it comes to installing PWAs. Yet it works with at least all chromium based browsers on mobile and desktop, but not with Firefox.

Discontinued, imho not a bug:
https://en.wikipedia.org/wiki/Progressive_web_app#Browser_support

tl;dr: It seems you're forced to use anything but Firefox 😐

**Comment by fr0tt** [comment]
I didn't expect that, but unfortunately that explains it.
